chore(curriculum): add colored markers to new cert (#56718)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d525007b8c5d8b3308b1c.md

---
id: 616d525007b8c5d8b3308b1c
title: Step 8
challengeType: 0
dashedName: step-8
---

# --description--

> Your marker would look better if it was centered on the page. An easy way to do that is with the `margin` <dfn>shorthand property</dfn>.

Reviewer: ```suggestion
Your marker would look better if it were centered on the page. An easy way to do this is by using the `margin` <dfn>shorthand property</dfn>.
```
